MinPlatformPkg/PlatformInit/PlatformInitPei/PlatformInitPreMem.c

```diff
@@ -186,6 +187,29 @@ BuildMemoryTypeInformation (
                                &DataSize,
                                &MemoryData
                                );
+
+  if (!EFI_ERROR(Status)) {

```

Comment:
While we're not running Uncrustify here, please add a space before opening parentheses throughout the code additions.

Reply:
Updated.